### PR TITLE
deleted Openads demo and connector_appxexus devDependencies

### PR DESCRIPTION
**Pull request by ReinerStarke**
- Deleted OpenAds demo.
- Deleted scripts related with demo
- Deleted dependencies to connector-appnexus

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,6 @@
     "prepublishOnly": "npm run cleanDistFolder && npm run build",
     "build": "npm run cleanDistFolder && babel src --ignore test,demo --out-dir dist ",
```
Comment (alextremp) [inline package.json]: remove "demo"

**Comment by victuxbb** [comment]
Also, check dependencies! "now" is not necessary now ;)
Anything related with webpack needs to be removed.

**Comment by victuxbb** [comment]
Remove now now!

File: package.json [hunk 2]
```diff
@@ -6,11 +6,7 @@
...
     "prepublishOnly": "npm run cleanDistFolder && npm run build",
-    "build": "npm run cleanDistFolder && babel src --ignore test,demo --out-dir dist ",
-    "deploy": "npm run cleanDistFolder && npm run demo:build && npm run now:upload",
-    "demo": "npm run cleanDistFolder && npm run demo:build && npm run server",
-    "demo:build": "webpack --config ./src/demo/webpack/webpack.config.js --progress --profile --colors --display-error-details --display-cached",
-    "now:upload": "now -n openads dist/",
+    "build": "npm run cleanDistFolder && babel src --ignore test --out-dir dist ",
     "test": "mocha --recursive --require babel-polyfill --compilers js:babel-register \"src/test/**/*.js\"",
     "test:integration": "mocha --recursive --require babel-polyfill --compilers js:babel-register \"src/itest/**/*.js\"",
     "server": "http-server './dist' -p 8080",
```
Comment (victuxbb) [inline package.json]: this is not necessary

File: package.json [hunk 3]
```diff
@@ -41,17 +36,12 @@
     "babel-register": "^6.24.0",
     "chai": "^4.0.2",
     "codecov": "^3.0.0",
-    "copy-webpack-plugin": "^4.2.3",
-    "html-webpack-plugin": "^2.30.1",
     "http-server": "^0.10.0",
```
Comment (victuxbb) [inline package.json]: same here

**Pull request merged**